proto/client/gClient.go

```diff
@@ -30,13 +31,13 @@ func init() {
 }
 
 type PlmnId struct {
-	MCC string
-	MNC string
+	Mcc string
+	Mnc string
 }
 
 type Nssai struct {
-	sst string
-	sd  string
+	Sst string
```

Comment:
Will this have impact across all NF repos?

Reply:
It should not because this is actually a "fix" because as you can see in https://github.com/omec-project/config5g/blob/main/proto/sdcoreConfig/config.pb.go#L433, their correct names are `Sst` and `Sd`. Actually, the "old" names (`sst` and `sd`) were reported as "unused"